iocage_lib/ioc_start.py

```diff
@@ -1424,38 +1422,45 @@ def __start_generate_vnet_mac__(self, nic):
 
         return mac_a, mac_b
 
-    def __check_dhcp_or_accept_rtadv__(self, dhcp):
+    def __check_dhcp_or_accept_rtadv__(self, ipv4, enable):
         # legacy behavior to enable it on every NIC
-        if dhcp and self.conf['dhcp']:
+        if ipv4 and (self.conf['dhcp'] or not enable):
```

Comment:
Don't you mean `or enable` here?

Reply:
Nope, in that case we want the `else` part to be executed because it means that user has custom specified DHCP on one of the entries in `ip4_addr`. This is to handle the case where DHCP is not desired